ccmlib/node.py

```diff
@@ -1295,6 +1302,41 @@ def stress(self, stress_options=None, capture_output=False, **kwargs):
         except KeyboardInterrupt:
             pass
 
+    @staticmethod
+    def _set_keyspace_initial_tablets(stress_options: List[str], initial_tablets:int = 8,
+                                      datacenter: Optional[str] = None, rf: int = 3) -> List[str]:
+        normalized_stress_options = []
+        for stress_option in stress_options:
+            # sometimes users pass schema params in one str, split it
+            if stress_option.startswith("-schema"):
+                if " " in stress_option:
+                    normalized_stress_options += stress_option.split()
+                    continue
+            normalized_stress_options.append(stress_option)
+        stress_options = normalized_stress_options
+        datacenter = datacenter or 'datacenter1'
+        default_replication = f"replication(strategy=NetworkTopologyStrategy,{datacenter}={rf},initial_tablets={initial_tablets})"
+        try:
+            idx = stress_options.index('-schema')
+            replication = stress_options[idx + 1]
+            if 'replication' in replication:
+                if 'NetworkTopologyStrategy' in replication:
+                    # if NTP already set, just prepend initial_tablets
+                    replication = re.sub(r'replication\(.+?\)',
+                                         lambda m: m.group(0).replace(')', f',initial_tablets={initial_tablets})'), replication)
+                else:
```

Comment:
shouldn't we check for `SimpleStrategy`?
what if it's `EverywhereStrategy`?
Maybe the funtion should throw an error in this case

Reply:
currently anything but NetworkStrategy will be overridden completely. I don't know much about EverywhereStrategy - is it available and can be used by user keyspaces?

Reply:
and more, can you pass initial_tablets on `EverywhereStrategy` and `SimpleStrategy` ?

Reply:
`SimpleStrategy` fails with `ConfigurationException: Unrecognized strategy option {initial_tablets} passed to org.apache.cassandra.locator.SimpleStrategy for keyspace keyspace1` 
`EverywhereStrategy` - I can create but it ignores tablets option (it is mentioned in describe keyspace, but `initial_tablets` don't change anything in sstables structure).

Reply:
EverywhereStrategy is used in 3 dtest modules only, non of them uses c-s, we should check if c-s even supports using it